zookeeper-api/src/main/java/org/apache/helix/zookeeper/zkclient/ZkClient.java

```diff
@@ -119,19 +115,6 @@
   // ZkEventThread. Otherwise the retry request might block the normal event processing.
   protected final ZkAsyncRetryThread _asyncCallRetryThread;
 
-  static {
```

Comment:
what is the difference of using this static block vs initalization at declaration?

Reply:
If you read the comment: // Set it here for unit test to use reflection to change value	
    // because compilers optimize constants by replacing them inline.

Reply:
I see.